src/symbols/rust_string.rs

```diff
@@ -19,6 +19,18 @@ unsafe extern "C" fn string_clone(this: &mut MaybeUninit<String>, other: &String
     unsafe { ptr::write(this, clone) }
 }
 
+#[export_name = "cxxbridge1$string$from_latin1"]
+unsafe extern "C" fn string_from_latin1(
+    this: &mut MaybeUninit<String>,
+    ptr: *const u8,
+    len: usize,
+) {
+    let slice = unsafe { slice::from_raw_parts(ptr, len) };
+    let this = this.as_mut_ptr();
+    let owned = slice.iter().map(|&c| c as char).collect();
```

Comment:
I'm not sure this is the best way to do it but it will work now. Could always optimize it later.